pkg/newlog/cmd/newlogd.go

```diff
@@ -287,6 +264,29 @@ func main() {
 		log.Fatal(err)
 	}
 
+	if vectorEnabled.Load() {
```

Comment:
Calling subGlobalConfig.Activate() doesn't mean that the handlers have been called. That requires doing the select on the channel and calling the handler (and checking that the Create handler was actually called).

Reply:
@OhmSpectator it's called inside `populate` when the subscription is persistent

Reply:
Actually, you may be right... I'll check once again

Reply:
Ok, the population will indeed put the config into the channel. But in theory, a race is still possible, as the handlers are called asynchronously to the placing of the message into the pipe. A more robust way would be something like we do in other agents: https://github.com/lf-edge/eve/blob/3ad0e22b42077bae739ec3f353b3a403368a2beb/pkg/pillar/cmd/zedagent/zedagent.go#L711
or any other component. Just grep for `GCInitialized`